modules/apps/jax-rs/mdm-admin/src/main/java/org/wso2/carbon/mdm/api/User.java

```diff
@@ -645,7 +638,6 @@ public Response inviteExistingUsersToEnrollDevice(List<String> usernames) throws
             int i;
             for (i = 0; i < usernames.size(); i++) {
```

Comment:
Move the int i declaration to inside the loop. This loop counter must be only visible inside the loop body.